tests/fixtures/analysis/taxprofiler/multiqc_data.json

```diff
@@ -0,0 +1,154 @@
+                "percent_gc": 42.0,
+                "avg_sequence_length": 150.31898468815814
+            }
+        },
+        {
+            "a_sample_name_a_sample_name": {
+                "filtering_result_passed_filter_reads": 24810472.0,
+                "filtering_result_low_quality_reads": 353472.0,
+                "filtering_result_too_many_N_reads": 362.0,
+                "filtering_result_too_short_reads": 9252.0,
+                "filtering_result_too_long_reads": 0.0,
+                "pct_duplication": 9.06289,
+                "after_filtering_total_reads": 24810472.0,
+                "after_filtering_total_bases": 3729762485.0,
+                "after_filtering_q20_bases": 3632227951.0
+            },
+            "another_sample_name_another_sample_name": {
+                "filtering_result_passed_filter_reads": 18583052.0,
+                "filtering_result_low_quality_reads": 357656.0,
+                "filtering_result_too_many_N_reads": 524.0,
+                "filtering_result_too_short_reads": 11144.0,
+                "filtering_result_too_long_reads": 0.0,
+                "pct_duplication": 5.29809,
+                "after_filtering_total_reads": 18583052.0,
+                "after_filtering_total_bases": 2780127919.0,
+                "after_filtering_q20_bases": 2691806490.0,
+                "after_filtering_q30_bases": 2544698121.0,
+                "after_filtering_q20_rate": 0.968231
+            }
+        },
+        {
+            "a_sample_name_a_sample_name_1": {
+                "Input reads": 21848064,
+                "Input bases": 3286972543,
+                "Low entropy discards reads": 11660,
+                "Low entropy discards reads percent": 0.05,
+                "Low entropy discards bases": 1722930
+            },
+            "another_sample_name_another_sample_1": {
+                "Input reads": 20388654,
+                "Input bases": 3049509473,
+                "Low entropy discards reads": 10294,
+                "Low entropy discards reads percent": 0.05,
+                "Low entropy discards bases": 1324808
+            }
+        },
+        {
+            "a_sample_name_a_sample_name": {
+                "total_reads": 12400055,
+                "paired_total": 12400055,
+                "paired_aligned_none": 1409340,
+                "paired_aligned_one": 9019738,
+                "paired_aligned_multi": 1970977,
+                "paired_aligned_discord_one": 679727,
+                "paired_aligned_mate_none": 593752,
+                "paired_aligned_mate_one": 374624,
+                "paired_aligned_mate_multi": 490850,
+                "overall_alignment_rate": 97.61,
+                "paired_aligned_mate_multi_halved": 245425.0,
+                "paired_aligned_mate_none_halved": 296876.0,
+                "paired_aligned_mate_one_halved": 187312.0
+            },
+            "another_sample_name_another_sample_name": {
```

Comment:
is this wanted? "another_sample_name_another_sample_name":

Reply:
Thanks for the review! In the `multiqc_data.json` file, the values of multiple samples within a case are reported and I wanted to make sure that the parsing is correct. Do you have any other suggestion of how this fixture can be?

Reply:
Yes, it is is intentional so I resolve this one.